charts/osm/crds/meshconfig.yaml

```diff
@@ -187,6 +187,14 @@ spec:
                       description: Sets the service certificate validity duration, represented as a sequence of decimal numbers each with optional fraction and a unit suffix.
                       type: string
                       default: "24h"
+                experimental:
+                  description: Experimental configurations
+                  type: object
+                  properties:
+                    clusterID:
+                      description: ID for this cluster; will be used by other clusters to reference it in a multi-cluster scenario.
+                      type: string
+                      default: "clusterID"
```

Comment:
will tis default actually be beneficial ?

Reply:
Fair enough, removed and set to omit if empty.